validator/nodejs/package.json

```diff
@@ -24,6 +24,6 @@
     "jasmine": "2.3.2"
   },
   "scripts": {
-    "postinstall": "/bin/sh -c \"exit 0\" 2> NUL && rm NUL || node postinstall-windows.js"
+    "postinstall": "/bin/sh -c \"exit 0\" 2> output && rm output || (del output && node postinstall-windows.js)"
```

Comment:
Rather than calling it 'output', may be better to make it more unique, like postinstall.DELETEME, to avoid that it clobbers something useful in the current working directory.
As for the del command, perhaps it's better to delete the file inside postinstall-windows.js? The del command will fail on non-Windows systems, so we could never get to the error message inside the script.

I think overall this solution is getting there, thanks a lot for looking into it.